### PR TITLE
Azo singleblog

### PR DESCRIPTION
**Pull request by vanikalai**
Single blog Require to merge with development

### CODE REVIEW

File: autozone-poc/src/components/Blog/Blog.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.blog-wrapper .single-blog-area .hover-content p{
+    height: 84px;
+    line-height: 2;
+    font-family: "Poppins", sans-serif;
+    font-size: 14px !important;
```
Comment (gokulwebdeveloper) [inline autozone-poc/src/components/Blog/Blog.css]: avoid using !important in the css

File: autozone-poc/src/components/Blog/Blog.js [hunk 2]
```diff
@@ -6,6 +6,8 @@ import blog4 from "../../utlis/img/bg-img/blog4.jpg";
 import blog5 from "../../utlis/img/bg-img/blog5.jpg";
 import blog6 from "../../utlis/img/bg-img/blog6.jpg";
```
Comment (gokulwebdeveloper) [inline autozone-poc/src/components/Blog/Blog.js:8]: Add propstype validation and default props to all components

File: autozone-poc/src/components/single-blog/Comments.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
+import React, { Fragment } from 'react';
+import '../single-blog/single-blog.css';
+import Button from '../../utlis/button/Button';
+
+const initialList = [];
+
+const Comments = () => {
+  const [value, setValue] = React.useState('');
```
Comment (gokulwebdeveloper) [inline autozone-poc/src/components/single-blog/Comments.js]: If possible merge all the state into a single state and handle the validation.

File: autozone-poc/src/components/single-blog/Comments.js [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+              <input type="text" placeholder="Email..." value={email} onChange={e => setEmail(e.target.value)}
+                className="form-control" />
+            </div>
+          </div>
+          <div className="form-group-textarea">
+            <textarea value={comments} onChange={e => setComments(e.target.value)}
+              placeholder="Comment..." cols="98" rows="8"></textarea>
+          </div>
+
+          {/* <input type="submit"
```
Comment (gokulwebdeveloper) [inline autozone-poc/src/components/single-blog/Comments.js]: Remove unwanted comment lines

File: autozone-poc/src/components/single-blog/single-blog.js [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+import Background from "../../utlis/img/bg-img/bg-7.jpg";
+import Rp1 from "../../utlis/img/bg-img/rp1.jpg";
+import Rp2 from "../../utlis/img/bg-img/rp2.jpg";
+import Rp3 from "../../utlis/img/bg-img/rp3.jpg";
+import Rp4 from "../../utlis/img/bg-img/rp4.jpg";
+import "../single-blog/single-blog.css";
+import Comments from "./Comments";
+
+//Component structure
+const Singleblog = (props) => {
```
Comment (gokulwebdeveloper) [inline autozone-poc/src/components/single-blog/single-blog.js:13]: Looks good make it as common utlis components. So others can also use throughout the website

**Comment by gokulwebdeveloper** [comment]
Please rebase all the commit

**Comment by gokulwebdeveloper** [comment]
http://tdongsi.github.io/blog/2016/07/05/git-allow-empty-when-squashing/

File: autozone-poc/src/components/Blog/Blog.css (hunk 1 above)
Comment (Sowmiya89) [reply autozone-poc/src/components/Blog/Blog.css]: Fixed

File: autozone-poc/src/components/single-blog/Comments.js (hunk 4 above)
Comment (vanikalai) [reply autozone-poc/src/components/single-blog/Comments.js]: Fixed

File: autozone-poc/src/components/single-blog/Comments.js (hunk 3 above)
Comment (vanikalai) [reply autozone-poc/src/components/single-blog/Comments.js]: Done

**Review by gokulwebdeveloper — APPROVED** [review]

**Pull request merged**